<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="c_time">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>121</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>8:22</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="date">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>121</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>星期日</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>10</y>
         <width>661</width>
         <height>471</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="main_theam">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>421</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>MAGIC MIRROR</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>20</y>
          <width>191</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="whether">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>71</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>icon/cloud.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="temperature">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>temp</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page1" native="true">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>351</width>
          <height>281</height>
         </rect>
        </property>
        <widget class="QLabel" name="labelGif">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>181</width>
           <height>151</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>gif1.gif</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
